munit/shared/src/main/scala-3.0.0-RC1/munit/internal/MacroCompat.scala

```diff
@@ -0,0 +1,52 @@
+package munit.internal
```

Comment:
what was the reason for this compat file? (out of curiosity)

Reply:
Also, we can probably remove the equivalent M2 file

Reply:
One small change `val source = value.asTerm.pos.sourceCode.getOrElse("")`

Maybe that is already thee in M3 ?

Reply:
it seems worth trying

Reply:
Ok, it seems to compile ok. Though I am still following up about the docs.